src/Php/PhpVersion.php

```diff
@@ -395,4 +395,9 @@ public function substrReturnFalseInsteadOfEmptyString(): bool
 		return $this->versionId < 80000;
 	}
 
+	public function supportsBcMathNumberOperatorOverloading(): bool
```

Comment:
I copied this from the original PR. But I'm not sure whether it's correct. I assume that the reason we even need this is because earlier PHP versions might use a polyfill (which wouldn't have the operators overloaded). But PHP 8.4 can also use a polyfill, because bcmath is an optional extension.

Reply:
I think it's okay. If someone uses operators with BCMath objects, we can safely assume they assume they have the extension.